public/docs/content/.pages.yml

```diff
@@ -1,7 +1,8 @@
 ---
 nav:
-  - Welcome: welcome
+  - Overview: overview
+  - Getting started: getting_started
```

Comment:
I think we should rename this to `Contributing` and move it below API Reference as this pertains to developers who want to contribute.

Reply:
recommendation: I'd separate the contributing from the `getting_started` if that's equally important. Absolutely ok to have it as a top level menu item, and another box on the landing page (the troubleshooting one will be gone anyways)